coverage/lcov-report/index.html

```diff
@@ -23,30 +23,30 @@ <h1>All files</h1>
         <div class='clearfix'>
             
             <div class='fl pad1y space-right2'>
-                <span class="strong">91.4% </span>
```

Comment:
add the coverage report in the .gitignore because everything is being passed through the test.